test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -284,24 +124,19 @@ public void wipeCluster() throws Exception {
         }
 
         // wipe index templates
-        Map<String, String> deleteTemplatesArgs = new HashMap<>();
-        deleteTemplatesArgs.put("name", "*");
-        adminExecutionContext.callApi("indices.delete_template", deleteTemplatesArgs, Collections.emptyList(), Collections.emptyMap());
+        adminClient().performRequest("DELETE", "_template/*");
```

Comment:
This kind of rough handling of the `Response` object is only ok after #19400 is merged, removing `Closeable` from the response.